src/bindings/python/.mypy.ini

```diff
@@ -0,0 +1,9 @@
+[mypy]
+python_version = 3.6
+files=src/bindings/python/flux
```

Comment:
Do we want mypy to run on the front-end commands in `src/cmd` too?

Reply:
Yeah, we probably do, and the tests also.  We do not want it to run on _flux though, I'll tweak that as well, thanks for noticing this.

Reply:
We now check almost everything, need a way to generate stubs for constants to get the last two files to type check.  I can generate these easily after a build of the python module by using the module itself, but getting it in place without building the full tree is much less straightforward.  Mulling over how to deal with this.